systests/transport-hc5/src/test/java/org/apache/cxf/systest/hc5/http/auth/DigestAuthTest.java

```diff
@@ -127,7 +127,7 @@ private Greeter setupClient(boolean async) throws Exception {
         client.setReceiveTimeout(600000);
         cond.setClient(client);
         if (async) {
-            if (cond instanceof AsyncHTTPConduit) {
+            if (cond.getClass().getName().endsWith("AsyncHTTPConduit")) {
```

Comment:
I think we should be specific here, the forceURLConnectionConduit should be "false", right? So the previous condition was correct:

```suggestion
if (cond instanceof AsyncHTTPConduit)
```

Reply:
Revised accordingly